libs/partners/ai21/tests/unit_tests/conftest.py

```diff
@@ -0,0 +1,74 @@
+from unittest.mock import Mock
+
+import pytest
+from ai21 import AI21Client
+from ai21.models import (
+    CompletionsResponse,
+    Completion,
+    CompletionData,
+    CompletionFinishReason,
+    ChatResponse,
+    Penalty,
+    ChatOutput,
+    RoleType,
+    FinishReason,
+)
+from pytest_mock import MockerFixture
+
+
+BASIC_DUMMY_LLM_PARAMETERS = {
```

Comment:
where did you get this `DUMMY` naming convention from ?

Reply:
No where in particular. If you don't like it, I can remove it